README.md

```diff
@@ -15,5 +15,5 @@ rpc-pass = 'password'        // elementsd rpc password
 rpc-user = 'user'            // elementsd rpc user
 service-bind = 'localhost'
 service-port = 8080
-wallets = 'wallet1,wallet2'  // liquid wallets
+service-units = ''           // systemd units
```

Comment:
better to give a real example here, because i can see that the string passed here "only" splits at `,`. So if one puts in `service-units = 'foo;bar'`, which looks ok, the exporter won't work correctly